server/tests/index.spec.js

```diff
@@ -28,3 +28,4 @@ import "./models/ratings.spec";
 import "./controller/follow/follow.spec";
 import "./controller/reaction/article.spec";
 import "./controller/reaction/comment.spec";
+// import "./controller/article/searchArticle.spec";
```

Comment:
Why is this here ?

Reply:
I kept that in a case in the future we switch test DB, it will not be forgotten to be added to the test runner